core/libcamera_app.cpp

```diff
@@ -773,6 +773,11 @@ libcamera::Stream *LibcameraApp::GetMainStream() const
 	return nullptr;
 }
 
+libcamera::CameraManager *LibcameraApp::GetCameraManager() const
```

Comment:
Perhaps this function should return a ``const libcamera::CameraManager *`` value?

Reply:
sure. as I said, my c++ is stuck in the late 90s...

Reply:
This ought to change to return ``const libcamera::CameraManager *``